docs/jdbc-doris-catalog.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Doris catalog"
+slug: /jdbc-doris-catalog
+keywords:
+- jdbc
+- Doris
+- metadata
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Gravitino provides the ability to manage Doris metadata.
```

Comment:
I would suggest change Doris to Apache Doris here to avoid ASF issue.

Reply:
Do you mean all `Doris` in the document, or just change at the beginning of the document?

Reply:
I think change at the beginning should be enough.